homeassistant/components/scrape/sensor.py

```diff
@@ -80,7 +80,7 @@ async def async_setup_entry(
     config = dict(entry.options)
     for sensor in config["sensor"]:
         sensor_config: ConfigType = vol.Schema(
-            TEMPLATE_SENSOR_BASE_SCHEMA.schema, extra=vol.ALLOW_EXTRA
+            TEMPLATE_ENTITY_BASE_SCHEMA.schema, extra=vol.ALLOW_EXTRA
```

Comment:
The schema is assumed to be valid, so we can bypass `TEMPLATE_SENSOR_BASE_SCHEMA` validation.
But we still need to initialise the templates.